examples/config.minimal.yml

```diff
@@ -7,5 +7,17 @@ backplane:
           value: "*"
         - name: "max-cores"
           value: "*"
+        - name: "persistentWorkerKey"
+          value: "*"
+        - name: "persistentWorkerCommand"
+          value: "*"
```

Comment:
We should switch to having the default minimal queue be:
```
properties:
  - name: "*"
```
Rather than extending with these.

Reply:
I'm not sure what you mean. Are you suggesting to replace `config.minimal.yml` with this?

```yaml
backplane:
  redisUri: "redis://localhost:6379"
  queues:
    - name: "cpu"
      properties:
        - name: "*"
worker:
  publicName: "localhost:8981"
  dequeueMatchSettings:
    allowUnmatched: true
```

Reply:
Yes. I'm only assuming that this works, but it looks like it should with the current code, and grants exactly the same level of acceptance as your previous change.